openpype/hosts/nuke/api/workfile_template_builder.py

```diff
@@ -988,6 +990,7 @@ def update_placeholder(*args):
         raise ValueError("Too many selected nodes")
 
     placeholder_item = placeholder_items[0]
-    window = WorkfileBuildPlaceholderDialog(host, builder)
+    window = WorkfileBuildPlaceholderDialog(host, builder,
+                                            parent=get_main_window())
```

Comment:
Shouldn't we change `exec_` to `show` in line `996` ?

Reply:
I was thinking that too, but for updating placeholders that felt a bit weird when I tested that.
The update placeholder is "update the selected placeholder" and it being modal and closing after updating felt totally reasonable. It felt weird if it'd remain open or if during the dialog open you'd start selecting other placeholders, etc.